fix(vm): cpu usage/idle handled by single worker (#12663)
emqx/emqx: changes/ce/fix-12663.en.md

> Fixed an issue where the sample `emqx_vm_cpu_use` and `emqx_vm_cpu_idle` in prometheus endpoint `/prometheus/stats` are always the values since system bootup.

Reviewer: ```suggestion
Fixed an issue where `emqx_vm_cpu_use` and `emqx_vm_cpu_idle` metrics in Prometheus endpoint `/prometheus/stats` are unchanged after system boot.
```

Author: Strictly speaking, the CPU usage is not "unchanged after system boot."
It's just that every time the usage is obtained, the calculated usage is the average usage "from system startup until the API is called."
For example:
8 hours after booting, the average usage during these 8 hours will be calculated.
One day after power-on, the average usage for that day will be calculated.
And the length of time has nothing to do with EMQX startup time.
